dev/ProgressBar/ProgressBar.xaml

```diff
@@ -47,50 +55,48 @@
                                 <VisualState x:Name="Indeterminate">
                                     <Storyboard RepeatBehavior="Forever">
                                         <DoubleAnimationUsingKeyFrames
-                                            Storyboard.TargetName="ProgressBarIndicator"
-                                            Storyboard.TargetProperty="(UIElement.RenderTransform).(CompositeTransform.TranslateX)"
-                                            Duration="0:0:1">
-                                            <EasingDoubleKeyFrame KeyTime="0" Value="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=TemplateSettings.ContainerAnimationStartPosition}">
-                                                <EasingDoubleKeyFrame.EasingFunction>
-                                                    <CubicEase/>
-                                                </EasingDoubleKeyFrame.EasingFunction>
-                                            </EasingDoubleKeyFrame>
-                                            <EasingDoubleKeyFrame KeyTime="0:0:1" Value="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=TemplateSettings.ContainerAnimationEndPosition}">
-                                                <EasingDoubleKeyFrame.EasingFunction>
-                                                    <CubicEase/>
-                                                </EasingDoubleKeyFrame.EasingFunction>
-                                            </EasingDoubleKeyFrame>
+                                            Storyboard.TargetName="IndeterminateProgressBarIndicator"
+                                            Storyboard.TargetProperty="(UIElement.RenderTransform).(CompositeTransform.TranslateX)">
+                                            <SplineDoubleKeyFrame KeyTime="0" Value="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=TemplateSettings.ContainerAnimationStartPosition}" KeySpline="0.4, 0.0, 0.6, 1.0" />
+                                            <SplineDoubleKeyFrame KeyTime="0:0:1.5" Value="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=TemplateSettings.ContainerAnimationEndPosition}" KeySpline="0.4, 0.0, 0.6, 1.0" />
+                                            <SplineDoubleKeyFrame KeyTime="0:0:2" Value="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=TemplateSettings.ContainerAnimationEndPosition}" KeySpline="0.4, 0.0, 0.6, 1.0" />
                                         </DoubleAnimationUsingKeyFrames>
+                                        <DoubleAnimationUsingKeyFrames
+                                            Storyboard.TargetName="IndeterminateProgressBarIndicator2"
```

Comment:
<DoubleAnimationUsingKeyFrames Storyboard.TargetProperty="(UIElement.RenderTransform).(TranslateTransform.X)" Storyboard.TargetName="ProgressBarIndeterminateIndicator2">
                                                <!-- Value = -150% -->
                                                <DiscreteDoubleKeyFrame KeyTime="0" Value="-600" />
                                                <!-- Value = -100% -->
                                                <DiscreteDoubleKeyFrame KeyTime="0:0:0.2" Value="-400"/>
                                                <!-- Value = 200% -->
                                                <SplineDoubleKeyFrame KeyTime="0:0:2.0" Value="800" KeySpline="0.4,0,0.6,1" />
                                            </DoubleAnimationUsingKeyFrames>
                                            <DoubleAnimationUsingKeyFrames Storyboard.TargetProperty="Opacity" Storyboard.TargetName="ProgressBarIndeterminateIndicator2">
                                                <DiscreteDoubleKeyFrame KeyTime="0" Value="0" />
                                                <DiscreteDoubleKeyFrame KeyTime="0:0:0.49" Value="0" />
                                                <DiscreteDoubleKeyFrame KeyTime="0:0:0.5" Value="1" />
                                                <DiscreteDoubleKeyFrame KeyTime="0:0:2.0" Value="1" />

Reply:
See comment above.

Also, specs (linked in comment above) specified -150% as start position, and 166.6% for end position.